### PR TITLE
Adds container logs

### PR DESCRIPTION
**Pull request by ashraffouda**
Signed-off-by: Ashraf Fouda <ashraf.m.fouda@gmail.com>

### CODE REVIEW

File: pkg/container/container.go [hunk 1]
```diff
@@ -297,6 +298,17 @@ func (c *Module) Run(ns string, data pkg.Container) (id pkg.ContainerID, err err
 	return pkg.ContainerID(container.ID()), nil
 }
 
+// Get logs of the container
+func (c *Module) Logs(ns string, containerID string) (logs string, err error) {
+	filePath := fmt.Sprintf("/var/cache/modules/contd/logs/%s/%s.log", ns, containerID)
```
Comment (muhamadazmy) [inline pkg/container/container.go]: No hard coded paths please. Nor Sprintf when concatinating paths. instead this should be 
```suggestion
	filePath := filepath.Join(c.root, ns, containerID)
```

File: pkg/qsfsd/qsfs.go [hunk 2]
```diff
@@ -130,6 +130,8 @@ func (q *QSFS) Mount(wlID string, cfg zos.QuantumSafeFS) (info pkg.QSFSInfo, err
 	)
 	if lerr := q.waitUntilMounted(ctx, mountPath); lerr != nil {
 		err = lerr
+		logs, _ := contd.Logs(ctx, qsfsContainerNS, wlID)
+		err = fmt.Errorf("%s\nContainer Logs\n%s", lerr.Error(), logs)
```
Comment (muhamadazmy) [inline pkg/qsfsd/qsfs.go]:
```suggestion
		err = errors.Wrapf(lerr, "Container Logs:\n%s", logs)
```

**Review by muhamadazmy — CHANGES_REQUESTED** [review]

File: pkg/qsfsd/qsfs.go (hunk 2 above)
Comment (ashraffouda) [reply pkg/qsfsd/qsfs.go]: updated

File: pkg/container/container.go (hunk 1 above)
Comment (ashraffouda) [reply pkg/container/container.go]: updated

**Review by muhamadazmy — APPROVED** [review]

**Pull request merged**